packages/server-admin-ui/src/views/appstore/appStore.scss

```
@@ -154,3 +154,381 @@
+
+.plugin-row__title-block {
+  flex: 1 1 auto;
+  min-width: 0;
+}
+
+.plugin-row__meta {
+  grid-area: meta;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 12px;
+  // Line up under the title text: 28px icon + 12px column gap = 40px.
```

Stylelint (17.9.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)